src/test/java/com/javachallengers/simpson/service/SimpsonCharacterServiceImplTest.java

```diff
@@ -0,0 +1,84 @@
+package com.javachallengers.simpson.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+import java.util.Optional;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import com.javachallengers.simpson.model.SimpsonCharacter;
+import com.javachallengers.simpson.repository.SimpsonCharacterRepository;
+
+/**
+ * Unit tests for {@link SimpsonCharacterServiceImpl}
+ */
+@Tag("UnitTest")
+class SimpsonCharacterServiceImplTest {
+	@Mock
+	private SimpsonCharacterRepository simpsonCharacterRepository;
+	
+	private SimpsonCharacterService simpsonCharacterService; 
+	
+	@BeforeEach
+	public void setup() {
+		MockitoAnnotations.initMocks(this);
+		this.simpsonCharacterService = new SimpsonCharacterServiceImpl(simpsonCharacterRepository);
+	}
+	
+	@Test
+	void getById_GIVEN_null_id_MUST_throw_exception() {
```

Comment:
use the same pattern as before aa_aaa_aaa and not aaa_AAA_aaa_aaa

Reply:
ok

Reply:
Renamed.